pkg/helpers/homebrew_test.go

```diff
@@ -0,0 +1,24 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPackageIsInCellar(t *testing.T) {
+	prefix, err := ioutil.TempDir("/tmp", "dad-brew")
```

Comment:
- is there a reason to specify the tmpdir? `ioutil.TempDir("", ...)`
- we should clean this folder up after the test.

But all that is annoying, let's use Filet (it's already used in this project): 
https://github.com/Flaque/filet#creating-temporary-directories